APISubmissionGuide.md

```diff
@@ -95,32 +95,19 @@
 ## FABS Submission Process
 
 ### Upload FABS File
-- Step 1: Call `/v1/upload_detached_file/` (POST) (Note, this is NOT the file upload)
-    - Header:
-        - `X-Session-ID`: string, session token id
+- Call `/v1/upload_detached_file/`
     - Payload:
```

Comment:
All of the technical documentation should be in `dataactbroker/README.md`, with just references in `APISubmissionGuide.md`.